subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/BuildScriptBuilder.java

```diff
@@ -103,76 +125,44 @@ public BuildScriptBuilder testRuntimeDependency(String comment, String... depend
         return dependency("testRuntime", comment, dependencies);
     }
 
-    /**
-     * Adds some arbitrary configuration to the _end_ of the build script.
-     */
-    public BuildScriptBuilder configuration(String comment, String config) {
-        this.config.add(new ConfigSpec(comment, Splitter.on("\n").splitToList(config)));
+    public BuildScriptBuilder taskMethodInvocation(String comment, String taskName, String taskType, String methodName) {
```

Comment:
I would generalize this. We're just invoking a method on a property.

Reply:
With the Kotlin DSL, a task is not a property of the project script target. You have to bring it into scope.